Resources/doc/config.rst

```diff
@@ -214,6 +214,39 @@ The following configuration shows a bunch of mapping examples:
             </doctrine_mongodb:config>
         </container>
 
+Custom types
+------------
+
+`Custom types`_ can come in handy when you're missing a specific mapping type
+or when you want to replace the existing implementation of a mapping type for
+your documents.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        doctrine_mongodb:
+            types:
+                # following two definitions are equivalent
```

Comment:
If I understand correctly, you're saying that the value can either be a FQCN string directly or an object with the FQCN string under a `class` property. I think it may be simpler to just present one style in the documentation example, though, unless there is a legitimate reason users would need to use the more verbose syntax (perhaps if other options beyond `class` were expected).

Reply:
Not right now, indeed it'll be better to just show simpler syntax 👍